examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,356 @@
+r"""
+
+.. _quantum_natural_gradient:
+
+Quantum natural gradient
+========================
+
+This example demonstrates the quantum natural gradient optimization technique
+for variational quantum circuits, originally proposed in
+`Stokes et al. (2019) <https://arxiv.org/abs/1909.02108>`__.
+
+Background
+----------
+
+The most successful class of quantum algorithms for use on near-term noisy quantum hardware
+is the so-called variational quantum algorithm. In variational quantum algorithms,
+a low-depth parametrized quantum circuit ansatz is chosen, and a problem-specific
+Hermitian observable measured. A classical optimization loop is then used to find
+the set of quantum parameters that *minimize* a particular measurement statistic
+of the quantum device. Examples of such algorithms include the :ref:`variational quantum
+eigensolver (VQE) <vqe>`, the `quantum approximant optimization algorithm (QAOA) <https://arxiv.org/abs/1411.4028>`__,
+and :ref:`quantum neural networks (QNN) <quantum_neural_net>`.
+
+Due to the inherent noise of near-term quantum hardware, most recent implementations

```

Comment:
I'd say more due to lack of imagination
```suggestion
Most recent implementations
```